src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas2bail/Cas2BailApplicationsTransformer.kt

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2bail
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ApplicationStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas2Application
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationSummaryEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.cas2bail.Cas2BailApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.cas2bail.Cas2BailApplicationSummaryEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.PersonInfoResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.NomisUserTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.PersonTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2.AssessmentsTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2.StatusUpdateTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2.TimelineEventsTransformer
+import java.util.*
+
+@Component("Cas2BailApplicationsTransformer")
+class Cas2BailApplicationsTransformer(
+  private val objectMapper: ObjectMapper,
+  private val personTransformer: PersonTransformer,
+  private val nomisUserTransformer: NomisUserTransformer,
+  private val statusUpdateTransformer: Cas2BailStatusUpdateTransformer,
+  private val timelineEventsTransformer: Cas2BailTimelineEventsTransformer,
+  private val assessmentsTransformer: Cas2BailAssessmentsTransformer,
+)  {
+
+
+  fun transformJpaToApi(jpa: Cas2BailApplicationEntity, personInfo: PersonInfoResult): Cas2Application {
+    return Cas2Application(
```

Comment:
just to confirm - we're reusing the Cas2Application API type for Cas2Bail?

Reply:
Currently yes but this will likely change